schema/me/follow_artists.js

```diff
@@ -0,0 +1,46 @@
+import gravity from '../../lib/loaders/gravity';
+import { map } from 'lodash';
+import { total as getTotal } from '../../lib/loaders/total';
+import Artist from '../artist';
+import {
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+} from 'graphql';
+
+const FollowArtists = new GraphQLObjectType({
+  name: 'FollowArtists',
+  fields: {
+    artists: {
+      type: new GraphQLList(Artist.type),
+      resolve: (data) => {
+        const artists = data.artists ? data.artists : data;
+        return map(artists, 'artist');
+      },
+    },
+    total_count: {
+      type: GraphQLInt,
```

Comment:
I'd prefer this to be nested schema called counts, with a numeral field for the specific set of counts, which would make it consistent with all the other schemas. But you're the consumer.